lib/features/space-tool/SpaceTool.js

```diff
@@ -282,15 +282,17 @@ SpaceTool.prototype.calculateAdjustments = function(elements, axis, delta, start
     var shapeStart = element[ axis ],
         shapeEnd = shapeStart + element[ AXIS_TO_DIMENSION[ axis ] ];
 
-    // shape to be moved
-    if ((delta > 0 && shapeStart > start) || (delta < 0 && shapeEnd < start)) {
+    // handle attachers separately
+    if (isAttacher(element)
+      && ((delta > 0 && getMid(element)[ axis ] > start)
+        || (delta < 0 && getMid(element)[ axis ] < start))) {
+      attachers.push(element);
 
-      // handle attachers separately
-      if (isAttacher(element)) {
-        attachers.push(element);
+      return;
+    }
 
-        return;
-      }
+    // shape to be moved
+    if ((delta > 0 && shapeStart > start) || (delta < 0 && shapeEnd < start)) {
```

Comment:
Actually works for me :tada:.

What is missing is decent test coverage for this case.